### DISCUSSION TITLE
阿里云盘通过davfs挂载到centos之后，列表可以正常访问，但访问内容时就直接报错

### DISCUSSION THREAD

**Discussion by fang141x** [AlistGo/alist#5102] [Q&A]
alist挂载到阿里云盘open之后，功能都通正常，但是通过davfs挂载到centos之后，列表可以正常访问，但访问内容时就直接报错，如下图：

![image](https://github.com/alist-org/alist/assets/1351630/854b263f-b04b-4960-9234-9efbb5cd9367)

**Comment by ywjdlq** [comment]
![image](https://github.com/alist-org/alist/assets/17327914/01a5bfa0-2c5a-4faf-8330-40813190b747)
davfs挂载,这里得改成本地代理

**Reply by fang141x** [reply]
谢谢，已解决问题了。

**Reply by ywjdlq** [reply]
有别的办法吗?我发的这种流量会经过本地,速度会有损耗
